packages/bundle-source/src/index.js

```diff
@@ -73,51 +74,79 @@ export default async function bundleSource(
     if (isEntry) {
       entrypoint = fileName;
     }
-    let unmappedCode = code;
+
+    // Parse the rolled-up chunk with Babel.
+    const ast = (babelParser.parse || babelParser)(code);
```

Comment:
What causes the type of `babelParser` to vary? Maybe leave a comment.

Reply:
Done.  Module import systems.